wiki/Article_styling_criteria/Formatting/en.md

```diff
@@ -768,7 +807,13 @@ The following is an example of what a table should look like:
 
 ## Blockquotes
 
-The blockquote is limited to quoting text from someone. It must not be used to format text otherwise.
+The blockquote is limited to quoting text from someone. It must not be used to format text otherwise. The name of the quoted person that follows the quote must be prepended by an em dash (`—`) as follows:
```

Comment:
Any suggestion for blockquote usage in translation? Should the original quote add alongside translated quote or not? Example:

Original
```
> At the current rate of return, it would be unlikely for me to purchase a personal jet within my lifetime, which is one of my primary life goals. Thus I held discussions with the team on alternate forms of monetisation, studying the current trends in similar free-to-play games currently available on the market.

—peppy, "osu!coins" [[33]][r]
```

translated
```
> Au taux de rendement actuel, il serait peu probable que je puisse acheter un jet personnel de mon vivant, ce qui est l'un de mes principaux objectifs de vie. J'ai donc discuté avec l'équipe des autres formes de monétisation possibles, en étudiant les tendances actuelles des jeux gratuits similaires actuellement disponibles sur le marché.

— peppy, "osu!coins" [[33]][r]
```

Reply:
i'd say a mention that it has been translated is enough, maybe in parentheses